planner/core/memtable_predicate_extractor_test.go

```diff
@@ -1659,11 +1661,18 @@ func TestPredicateQuery(t *testing.T) {
 	tk := testkit.NewTestKit(t, store)
 	tk.MustExec("use test")
 	tk.MustExec("create table t(id int, abclmn int);")
-
 	tk.MustQuery("select TABLE_NAME from information_schema.columns where table_schema = 'test' and column_name like 'i%'").Check(testkit.Rows("t"))
 	tk.MustQuery("select TABLE_NAME from information_schema.columns where table_schema = 'TEST' and column_name like 'I%'").Check(testkit.Rows("t"))
 	tk.MustQuery("select TABLE_NAME from information_schema.columns where table_schema = 'TEST' and column_name like 'ID'").Check(testkit.Rows("t"))
 	tk.MustQuery("select TABLE_NAME from information_schema.columns where table_schema = 'TEST' and column_name like 'id'").Check(testkit.Rows("t"))
 	tk.MustQuery("select column_name from information_schema.columns where table_schema = 'TEST' and (column_name like 'I%' or column_name like '%D')").Check(testkit.Rows("id"))
 	tk.MustQuery("select column_name from information_schema.columns where table_schema = 'TEST' and (column_name like 'abc%' and column_name like '%lmn')").Check(testkit.Rows("abclmn"))
+	tk.MustQuery("describe t").Check(testkit.Rows("id int(11) YES  <nil> ", "abclmn int(11) YES  <nil> "))
+	tk.MustQuery("describe t id").Check(testkit.Rows("id int(11) YES  <nil> "))
+	tk.MustGetErrCode("describe t 'I%'", errno.ErrParse)
+	tk.MustGetErrCode("describe t I%", errno.ErrParse)
+
+	tk.MustQuery("show columns from t like `ABCLMN`").Check(testutil.RowsWithSep(",", "abclmn,int(11),YES,,<nil>,"))
+	tk.MustQuery("show columns from t like 'abc%'").Check(testutil.RowsWithSep(",", "abclmn,int(11),YES,,<nil>,"))
+	tk.MustQuery("show columns from t like '%lmn'").Check(testutil.RowsWithSep(",", "abclmn,int(11),YES,,<nil>,"))
```

Comment:
Please have more test cases according to https://docs.pingcap.com/zh/tidb/stable/sql-statement-show-columns-from/, such as
- show columns from t where field='id';
- show fields in t where field like 'abc%';

Reply:
Now, ``` show XXXX like XXXX``` has been implemented. So ``` where``` is not supported. I will add the ```rewrite``` to implement it for ```where```.

Reply:
`where` is already supported.

Reply:
> `where`

You say the query is from ```information_schema.columns``` . but the implementation of ShowStmt  is difference from ```memtable``` in the ```information_schema.columns```. I have tested the ``` where```. it is not case-insensitivity

Reply:
I mean `show columns from t1 where field='s1';`

Reply:
OK, I can add it.

Reply:
Done